### ISSUE TITLE
Tab order in images edit panel/dialog doesn't work as expected.

### ISSUE THREAD

**Issue by jswiderski** [ckfinder/ckfinder#160]
Start tabbing through CKFinder.

You should be able to move from toolbar to editor options and then perhaps to main panel (for Kadrowanie :)).

At present order is:
IE: Toolabr, bottom bar, main panel, ??? somewhere, edit options, links on main page.
FF Toolbar, main panel, links on main page.

__It is possible to focus links on main page which "is behind" edit panel/dialog__

**Comment by jodator** [comment]
I think that this works now ok. There are no gaps when tabbing.

**Comment by wwalc** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
